cmd/create.go

```diff
@@ -410,8 +412,8 @@ func createHost(provider, name, region, zone, projectID, userData, inletsPort st
 	} else if provider == "civo" {
 		return &provision.BasicHost{
 			Name:       name,
-			OS:         "811a8dfb-8202-49ad-b1ef-1e6320b20497",
-			Plan:       "g2.small",
+			OS:         CivoUbuntu2004ID,
```

Comment:
Inline the variable and use a lowercase C please. This doesn't need global package scope.

```go
	} else if provider == "civo" {
               const ubuntu2004ID = "d927ad2f-5073-4ed6-b2eb-b8e61aef29a8"
 		return &provision.BasicHost{
```